test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -14,6 +15,16 @@ public void setUp()
     }
 
     @Test
+    public void constructorReadsPropertiesFile()
+    {
+      WasapiDownloader myInstance = new WasapiDownloader();
+      assertNotNull(myInstance.settings);
+      assertEquals("incorrect default settings.properties baseurl value", myInstance.settings.getProperty("baseurl"), "http://example.org");
+      assertEquals("incorrect default settings.properties username value", myInstance.settings.getProperty("username"), "user");
+      assertEquals("incorrect default settings.properties password value", myInstance.settings.getProperty("password"), "pass");
+    }
+
```

Comment:
i think the testing is fine as-is.  it doesn't test the ins and outs of the private method, but it tests that the things are in the state we expect after calling the constructor, which calls the private method (and while i'm not against testing private methods in general, it's not really possible in Java the way it is in Ruby, AFAIK).